test/Shared/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
+        /// <summary>
+        ///     Navigation property for users in the role
+        /// </summary>
+        public virtual ICollection<IdentityUserRole<TKey>> Roles { get; private set; } = new List<IdentityUserRole<TKey>>();
```

Comment:
so the navigation properties are needed since the InMemory tests makes use of them. I need to keep them